pdr_backend/pdr_dashboard/util/helpers.py

```diff
@@ -64,12 +64,17 @@ def select_or_clear_all_by_table(
 
 @enforce_types
 def _format_date_text(
-    start_date: Optional[datetime], end_date: Optional[datetime]
+    start_date: Optional[datetime],
+    end_date: Optional[datetime],
+    show_end_date_time: bool = False,
 ) -> str:
     if not start_date or not end_date:
         return "there is no data available"
 
-    return f"{start_date.strftime('%d-%m-%y')} -> {end_date.strftime('%d-%m-%y')}"
+    return f"""
+        {start_date.strftime('%d-%m-%y')} -> {end_date.strftime('%d-%m-%y')}
```

Comment:
only end date gets the time part?

Reply:
I think the start date is not relevant since it is the same hour each time only the date changes, and only when ETL is run for the first time. The end date is constantly changing and it's much more relevant with the incremental update because is constantly changing